src/PAModel/Utility/Utilities.cs

```diff
@@ -453,5 +453,12 @@ public static void VerifyDirectoryExists(ErrorContainer errors, string fullpath)
                 errors.BadParameter($"Directory not found: {fullpath}");
             }
         }
+
+        public static bool IsLocalDatabaseReferencesEmpty(this Entropy entropy)
+        {
+            return (entropy.WasLocalDatabaseReferencesEmpty != null) ?
+                   (bool)entropy.WasLocalDatabaseReferencesEmpty:
```

Comment:
For nullable properties, use `.Value` to retrieve the value, and `.HasValue` to test instead of comparing to null.

Reply:
Moved this method to Entropy.cs. Refactored this condition into null coalescing